GDJS/Runtime/SpriteAnimator.ts

```diff
@@ -123,6 +123,9 @@ namespace gdjs {
       frameData: SpriteFrameData,
       textureManager: gdjs.AnimationFrameTextureManager<T>
     ) {
+      this.image = frameData.image;
+      this.texture = textureManager.getAnimationFrameTexture(this.image);
```

Comment:
Why is this useful?

Reply:
A change of resource of a sprite was not updated by the hot-reload otherwise.